### PR TITLE
Update configure-persistent-volume-storage.md

### PR DESCRIPTION
**Pull request by currycan**

### CODE REVIEW

**Comment by tanjunchen** [comment]
Thanks for your job!
@currycan 
pls see https://git.k8s.io/community/CLA.md#the-contributor-license-agreement

File: content/zh/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md [hunk 1]
```diff
@@ -131,7 +131,7 @@ Create the PersistentVolume:
 -->
 
 配置文件指定了该卷位于集群节点上的 `/mnt/data` 目录。
-该配置还指定了 10 吉比特的卷大小和 `ReadWriteOnce` 的访问模式，这意味着该卷可以在单个节点上以读写方式挂载。
+该配置还指定了 10G 的卷大小和 `ReadWriteOnce` 的访问模式，这意味着该卷可以在单个节点上以读写方式挂载。
```
Comment (tanjunchen) [inline content/zh/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md:134]: 10G 的卷大小 -> 10 G 的卷大小

**Comment by markthink** [comment]
@currycan 参考[此文档](https://github.com/k8smeetup/k8s-official-translation)

**Comment by tengqm** [comment]
Closing this due to CLA status failure for more than one month. Please feel free to reopen it when the CLA issue is fixed.
/close

**Pull request closed without merging**